qubesmanager/qube_manager.py

```diff
@@ -969,27 +998,20 @@ def change_network(self, netvm_name):
         if reply != QMessageBox.StandardButton.Yes:
             return
 
-        if netvm_name:
-            check_power = any(info.state['power'] == 'Running' for info
-                    in self.get_selected_vms())
-            if netvm_name == 'default':
-                netvm = self._get_default_netvm()
-            else:
-                netvm = self.qubes_cache.get_vm(name=netvm_name)
-                netvm = netvm.vm
-            if check_power and netvm and not netvm.is_running():
-                reply = QMessageBox.question(
-                    self, self.tr("Qube Start Confirmation"),
-                    self.tr("<br>Can not change netvm to a halted Qube.<br>"
-                        "Do you want to start the Qube <b>'{0}'</b>?").format(
-                        netvm_name),
-                    QMessageBox.StandardButton.Yes |
-                    QMessageBox.StandardButton.Cancel)
+        # pylint: disable=possibly-used-before-assignment
+        if netvm_name and check_power and netvm and not netvm.is_running():
```

Comment:
This can be simplified to avoid checking both `netvm_name` and `netvm`, you just need to initialize `netvm = None` at the start of the function. This will also avoid the pylint complain (you may also need to initialize `check_power` too).